NPM-2.md

```diff
@@ -0,0 +1,132 @@
+## Preamble
+
+  NMP: NMP-2
+
+  Title: Carbon Removal Claims (CRCs) for Agricultural Soils (Croplands, Grazing and Rangelands) 
+
+  Extended Title: Soil Methodologies: CRC Supplier Pre-Listing And Preliminary Soil CRC Rating Requirements
+
+  Author: Aldyen Donnelly
+
+  Type: Methodology
+
+  Categroy: Carbon Removal
+
+  Status: Draft
+
+  Created: February 22, 2018
+
+  Replaces: N/A
+
+## Soil CRC Supplier Pre-Listing And Preliminary Soil CRC Rating Requirements
+
+### Summary
+
+This methodology outlines the actions a potential Carbon Removal Claim ("CRC")
+Supplier must take to register a Soil based CRC-generating Project on the Nori platform.
+Once a Project is registered on the platform, the designated "Responsible
+Person" (the "Supplier") is permitted to submit a claim to Nori,
+instructing the Nori platform administrator to list verified CRCs generated by
+the Registered Project for sale on the Nori platform for up to 10 years, subject
+to complying with NMP-2, Recurring Annual Reporting Requirements.  During Year
+10, the Supplier may re-apply for Project Registration for a subsequent 10-year
+term.  Provide a simplified explanation of the project/methodology being
+proposed.
+
+### Establishing Project/CRC Ownership & the "Responsible Person"
```

Comment:
Why require a US-based person to act as agent? Is this a commodities regulatory requirement?

Reply:
I'll let @Aldyen answer about the regulation part, but my understanding is that it reduces our risks as market operator because there will be a US resident subject to enforcement of US laws in the event of fraud or other nefarious activity. The other aspect is that this is standard practice for all commodities that are traded in US based markets, including things like oil and gas.